### ISSUE TITLE
GDScript Autocomplete for OS not showing writing if statements

### ISSUE THREAD

**Issue by DleanJeans** [godotengine/godot#26560]
**Godot version:** Godot 3.1 Beta 10

**OS/device including version:** Windows 10

**Issue description** / **Steps to reproduce:**

```
func _ready():
    OS. # works here
    if OS. # no autocomplete here
```
Not showing even when pressing Ctrl+Space

Edit: Same with `Engine`

**Comment by avencherus** [comment]
Likewise.  Usually have to complete the line, then go back to it to trigger the hints.

**Comment by breakmt** [comment]
Related #26114

**Comment by Chaosus** [comment]
I closed this as a duplicate of #26114 but renames the original accordingly(to all singletons)
